WrathCombo/Combos/PvE/GNB/GNB.cs

```diff
@@ -2026,17 +2030,6 @@ protected override uint Invoke(uint actionID, uint lastComboMove, float comboTim
                     //oGCDs
                     if (CanWeave())
                     {
-                        //Variant SpiritDart
-                        Status? sustainedDamage = FindTargetEffect(Variant.Debuffs.SustainedDamage);
-                        if (IsEnabled(CustomComboPreset.GNB_Variant_SpiritDart) &&
-                            IsEnabled(Variant.VariantSpiritDart) &&
-                            (sustainedDamage is null || sustainedDamage?.RemainingTime <= 3))
-                            return Variant.VariantSpiritDart;
```

Comment:
Confused about the removal of these Variant spell usages, especially when the options are still there in the `CustomComboPreset.cs` file.
Do they not belong in this particular combo, or something?